### PR TITLE
v3.5 OpenShift metrics recommendations

### PR DESCRIPTION
**Pull request by ekuric**
This is OpenShift v3.5 metrics recommendations with improvements delivered in v3.5  
Signed-off-by: Elvir Kuric <elvirkuric@gmail.com>

### CODE REVIEW

**Comment by ekuric** [comment]
@jeremyeder @mffiedler please check and propose changes where necessary

File: admin_guide/metrics_recommendations.adoc [hunk 1]
```diff
@@ -0,0 +1,30 @@
+== OpenShift v3.5 metrics recommendations
+
+In OpenShift v3.4 metrics https://docs.openshift.org/latest/install_config/cluster_metrics.html[documentation]
+are presented results which changed to better in OpenShift v3.5 metrics
+
```
Comment (mffiedler) [inline admin_guide/metrics_recommendations.adoc]: Suggestion:  "The OpenShift v3.4 metrics documentation presents results which have improved in OpenShift v3.5 metrics"

File: admin_guide/metrics_recommendations.adoc [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+The reason for better results lies in change of `METRIC_RESOLUTION` parameter
```
Comment (mffiedler) [inline admin_guide/metrics_recommendations.adoc]: Suggestion:  "in the change of the default value of the METRICS_RESOLUTION parameter

File: admin_guide/metrics_recommendations.adoc [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+in OpenShift metrics v3.5. New value for `METRICS_RESOLUTION` is 30 (seconds) and
```
Comment (mffiedler) [inline admin_guide/metrics_recommendations.adoc]: "The new default value"

File: admin_guide/metrics_recommendations.adoc [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+that change was introduced in order to better match cAdvisor housekeeping
```
Comment (mffiedler) [inline admin_guide/metrics_recommendations.adoc]: "the cAdvisor housekeeping"

File: admin_guide/metrics_recommendations.adoc [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+interval of 30 seconds. More details can be found in https://bugzilla.redhat.com/show_bug.cgi?id=1421834[Bugzilla]
+
+Increasing `METRICS_RESOLUTION` interval in OpenShift v3.5 helped to get better results in relation
+how many pods is possible to monitor by one set of metrics pods. Tests showed that OpenShift metrics
```
Comment (mffiedler) [inline admin_guide/metrics_recommendations.adoc]: "In OpenShift v3.5, tests showed that OpenShift metrics collection was stable for test cases when up to 25000"

File: admin_guide/metrics_recommendations.adoc [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+worked stable for test cases when up 25000 pods were monitored in OpenShift Container Platform Cluster.
+
+General recommendations for OpenShift Metrics v3.5 can be summarized to below
```
Comment (mffiedler) [inline admin_guide/metrics_recommendations.adoc]: "can be summarized as:"

File: admin_guide/metrics_recommendations.adoc [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+
+- running metrics pods on dedicated OpenShift nodes
```
Comment (mffiedler) [inline admin_guide/metrics_recommendations.adoc]: "run metrics pods on"

File: admin_guide/metrics_recommendations.adoc [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+- use persistent storage when configuring metircs `USE_PERSISTENT_STORAGE=true`
```
Comment (mffiedler) [inline admin_guide/metrics_recommendations.adoc]: typo -> metrics

File: admin_guide/metrics_recommendations.adoc [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+- keep `METRICS_RESOLUTION=30` parameter in OpenShift metrics deployments. It is recommended not to use lower
```
Comment (mffiedler) [inline admin_guide/metrics_recommendations.adoc]: Add:  When using the Ansible metrics installation procedure, this is the openshift_metrics_resolution parameter.

File: admin_guide/metrics_recommendations.adoc [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+value than default 30 for `METRICS_RESOLUTION`
+- closely monitor OpenShift nodes which host metrics pods in order to detect early capacity
```
Comment (mffiedler) [inline admin_guide/metrics_recommendations.adoc]: "OpenShift nodes with host metrics pods"

**Review by mffiedler — COMMENTED** [review]
No technical issues - just some editorial/flow suggestions.

**Comment by ahardin-rh** [comment]
@ekuric  I am closing this in favor of https://github.com/openshift/openshift-docs/pull/4063 and https://github.com/openshift/openshift-docs/pull/4046. I forked your branch and continued this work there.

cc @mffiedler @jeremyeder

**Pull request closed without merging**